### DISCUSSION TITLE
Disconnected

### DISCUSSION THREAD

**Discussion by Tieum3564** [Thomas55555/husqvarna_automower#183] [Q&A]
Hello, I use this custom component for few weeks now. It worked like a charm but I'm continuously disconnected since the last Home Assistant update to 2022.03.6. I don't know if it's linked or not. Have you heard the same from other users? Is there a fix for this issue? Thanks.

**Comment by Thomas55555** [comment]
If there are no status updates from the websocket anymore, then the mower changes to unavailable, see #186
My goal with this was to find out, if the mower is turned off for example. Right now the websocket is not working anymore. So the mower also changes to unavailable. Because you wouldn't receive state updates anymore. So it's not that wrong.
I don't have a proper solution for that. @simontegelid: Do you have an idea?

**Reply by Tieum3564** [reply]
Now I know where the issue come from. Thank you for your answer. Hope it will be fixed. It's a really good integration and I would like to keep using it.

**Comment by Thomas55555** [comment]
I think they only have to restart the service on their server. But I don't write an email to them, because this happens every two month. I hope @simontegelid will have a look on it first. So that the integration can detect if the websocket service is not working anymore. So that in the future the integration isn't blamed for it if the websocket isn't working anymore.
